x-pack/filebeat/module/suricata/eve/test/eve-alerts.log-expected.json

```diff
@@ -632,15 +642,16 @@
         "destination.geo.region_name": "England", 
         "destination.ip": "91.189.88.152", 
         "destination.port": 80, 
+        "ecs.version": "1.0.0-beta2", 
         "event.dataset": "eve", 
         "event.module": "suricata", 
         "event.type": "alert", 
         "http.request.method": "GET", 
         "http.response.status_code": "200", 
         "input.type": "log", 
         "log.offset": 7630, 
-        "source.ip": "192.168.1.146",
-        "source.port": 37742,
+        "source.ip": "192.168.1.146", 
```

Comment:
@ycombinator Interesting, this now also happens in my PRs that spaces are added?!?